pkg/controllers/deprovisioning/emptymachineconsolidation.go

```diff
@@ -43,6 +43,7 @@ func NewEmptyMachineConsolidation(clk clock.Clock, cluster *state.Cluster, kubeC
 // ComputeCommand generates a deprovisioning command given deprovisionable machines
 func (c *EmptyMachineConsolidation) ComputeCommand(ctx context.Context, candidates ...*Candidate) (Command, error) {
 	if c.cluster.Consolidated() {
+		logging.FromContext(ctx).Debugf("skipping empty node consolidation since cluster is consolidated")
```

Comment:
This one seems to provide marginal value, while being quite spammy. If anything, can we log on the execution of each deprovisioner in controller.go?